### PR TITLE
Refactor Exceptions and remove unimportant variables and logs

### PR DESCRIPTION
**Pull request by anjumfatima90**

### CODE REVIEW

File: boost-maven-plugin/src/main/java/io/openliberty/boost/docker/Dockerize.java [hunk 1]
```diff
@@ -86,7 +86,7 @@ private void createSpringBootDockerFile(String springBootVersion) throws MojoExe
     }
     
     @SuppressWarnings("resource")
-    private boolean isFileExecutable(File file) throws IOException {
+    private boolean isFileExecutable(File file) throws IOException, MojoExecutionException {
         if (file.exists()) {
```
Comment (chyt) [inline boost-maven-plugin/src/main/java/io/openliberty/boost/docker/Dockerize.java]: Add more checks for the file, i.e. `if(file != null && file.exists() && file.isFile())`

File: boost-maven-plugin/src/main/java/io/openliberty/boost/docker/Dockerize.java [hunk 2]
```diff
@@ -86,7 +86,7 @@ private void createSpringBootDockerFile(String springBootVersion) throws MojoExe
     }
     
     @SuppressWarnings("resource")
-    private boolean isFileExecutable(File file) throws IOException {
```
Comment (chyt) [inline boost-maven-plugin/src/main/java/io/openliberty/boost/docker/Dockerize.java]: Can we just have it throw a `MojoExecutionException`? I would catch the `IOException` and throw a `MojoExecutionException` in that case, passing the `IOException` through with an error message relevant to this method.

File: boost-maven-plugin/src/main/java/io/openliberty/boost/docker/DockerBuildMojo.java [hunk 3]
```diff
@@ -71,36 +71,40 @@ protected void execute(DockerClient dockerClient) throws MojoExecutionException,
...
     private void buildDockerImage(DockerClient dockerClient, File appArchive)
             throws MojoExecutionException, IOException {
-        final ArrayList<BuildParam> buildParameters = new ArrayList<>();
-        final BuildParam[] buildParametersArray;
         final DockerLoggingProgressHandler progressHandler = new DockerLoggingProgressHandler(log);
         final String imageName = getImageName();
+        BuildParam[] buidParams = getBuildParams(appArchive);
 
+        log.info("Image will be built as " + imageName);
+        log.info("");
```
Comment (chyt) [inline boost-maven-plugin/src/main/java/io/openliberty/boost/docker/DockerBuildMojo.java]: Why have an empty log line here?

Comment (anjumfatima90) [reply boost-maven-plugin/src/main/java/io/openliberty/boost/docker/DockerBuildMojo.java]: fixed.

File: boost-maven-plugin/src/main/java/io/openliberty/boost/docker/Dockerize.java (hunk 2 above)
Comment (anjumfatima90) [reply boost-maven-plugin/src/main/java/io/openliberty/boost/docker/Dockerize.java]: DockerBuildMojo.execute() is catching Exception and throwing a MojoExecutionException

File: boost-maven-plugin/src/main/java/io/openliberty/boost/docker/Dockerize.java (hunk 1 above)
Comment (anjumfatima90) [reply boost-maven-plugin/src/main/java/io/openliberty/boost/docker/Dockerize.java]: fixed

**Pull request merged**